config/vufind/Folio.ini

```diff
@@ -106,13 +106,19 @@ extraHoldFields = requiredByDate:pickUpLocation
 ;default_request = Hold
 
 ; Hide the place hold/recall/page link when an item is in the configured
-; list of locations. Ideally, this would not be needed and FOLIO would include an
+; list of locations. This can either match the full location name exactly or
+; do a regex match depending on how you set the excludeHoldLocationsCompareMode
+; setting. Ideally, this would not be needed and FOLIO would include an
 ; indicator of whether or not an item is holdable with item details.
 ; See https://issues.folio.org/browse/UXPROD-2422.
 ; Checks against the location's Discovery Display Name in FOLIO.
 ; If no Discovery Display Name is configured, checks against name.
+;excludeHoldLocationsCompareMode = exact
```

Comment:
Might be worth explicitly listing legal values in a comment here -- the comment above mentions this setting but isn't totally explicit about what settings are supported (e.g. `; Legal excludeHoldLocationsCompareMode settings: exact, regex; see examples below.`).